schema/vr.yaml

```diff
@@ -237,5 +267,5 @@ definitions:
       Using internal ids in public messages is strongly discouraged.
 
     type: string
-    pattern: '^\w[^:]+:.+$'
-    example: "ga4gh:GA.01234abcde"
+    pattern: '^\w[^:]*:.+$'
```

Comment:
See above for comment on the regex, final resolution should be implemented in both places for consistency

Reply:
+1 - and had already implemented it so in a small mapping service I'll link soonish...

Reply:
My proposed change to the regex (above) was incorrect. I misinterpreted the `\w` as leading whitespace, when I believe it represented a "word" character. The regex pattern does not permit leading or trailing white space, but the proposed version (`pattern: '^\w[^:]*:.+$'`) is correct (provided CURIEs really do permit any character in the reference portion, as indicated by the `.+`).

Reply:
True; there are few one-character prefixes :-)

Reply:
I'm fine with permitting 1-char prefixes.

Reply:
@reece So identifiers.org doesn't even support searching single-letter ones; me myself and I just tried - no hope for us getting `I:`, I guess.

Reply:
Did you try :star: ?

Because bioinformatics really needs unicode namespaces to improve interoperability. :-/